app/src/main/java/com/example/mycalculator/mvp/CalculatorModelImpl.java

```diff
@@ -0,0 +1,17 @@
+package com.example.mycalculator.mvp;
+
+public class CalculatorModelImpl implements CalculatorContract.CalculatorModelContract {
+
+    private String firstOperator;
+    private String secondOperator;
```

Comment:
you are not using this variable, pls remove it

Reply:
Done. Thx!